src/wrappers/any.jl

```diff
@@ -115,3 +116,87 @@ Base.show(io::IO, status::Status) = print(io, ToString(status))
+    buffer_size = Size(buffer[])
+    vec = Vector{UInt8}(undef, buffer_size)
+    unsafe_copyto!(Ptr{UInt8}(pointer(vec)), buffer_ptr, buffer_size)
+    return vec
+end
+
+function task_executor()
+    @info "task_executor called"
+    return getpid()
+end
+
+project_dir() = dirname(Pkg.project().path)
+submit_task() = _submit_task(project_dir())
+
+
+#=
+julia -e sleep(120) -- \
+  /Users/cvogt/.julia/dev/ray_core_worker_julia_jll/venv/lib/python3.10/site-packages/ray/cpp/default_worker \
+  --ray_plasma_store_socket_name=/tmp/ray/session_2023-08-09_14-14-28_230005_27400/sockets/plasma_store \
+  --ray_raylet_socket_name=/tmp/ray/session_2023-08-09_14-14-28_230005_27400/sockets/raylet \
+  --ray_node_manager_port=57236 --ray_address=127.0.0.1:6379 \
+  --ray_redis_password= \
+  --ray_session_dir=/tmp/ray/session_2023-08-09_14-14-28_230005_27400 \
+  --ray_logs_dir=/tmp/ray/session_2023-08-09_14-14-28_230005_27400/logs \
+  --ray_node_ip_address=127.0.0.1
+=#
+function start_worker(args=ARGS)
+    s = ArgParseSettings()
+
+    # Worker options are generated in the Raylet function `BuildProcessCommandArgs`
+    # (https://github.com/ray-project/ray/blob/ray-2.5.1/src/ray/raylet/worker_pool.cc#L232)
+    # and are parsed in Python here:
+    # https://github.com/ray-project/ray/blob/ray-2.5.1/python/ray/_private/workers/default_worker.py
+    @add_arg_table! s begin
+        "--ray_raylet_socket_name"
+            dest_name = "raylet_socket"
+            arg_type = String
+        "--ray_plasma_store_socket_name"
+            dest_name = "store_socket"
+            arg_type = String
+        "--ray_address"  # "127.0.0.1:6379"
+            dest_name = "address"
+            arg_type = String
+        "--ray_node_manager_port"
+            dest_name = "node_manager_port"
+            arg_type = Int
+        "--ray_node_ip_address"
+            dest_name = "node_ip_address"
+            required=true
+            arg_type=String
+            help="The ip address of the worker's node"
+        "--ray_redis_password"
+            dest_name = "redis_password"
+            required=false
+            arg_type=String
+            help="the password to use for Redis"
+        "--ray_session_dir"
+            dest_name = "session_dir"
+        "--ray_logs_dir"
+            dest_name = "logs_dir"
+        "--runtime-env-hash"
+            dest_name="runtime_env_hash"
+            required=false
+            arg_type=Int
+            default=0
+            help="The computed hash of the runtime env for this worker"
+        "arg1"
+            required = true
+    end
+
+    parsed_args = parse_args(args, s)
+
+    open(joinpath(parsed_args["logs_dir"], "julia_worker.log"), "w+") do io
+        global_logger(SimpleLogger(io))
+        @info "Testing"
+        initialize_coreworker_worker(
+            parsed_args["node_manager_port"],
+            CxxWrap.@safe_cfunction(task_executor, Int32, ()),
+        )
+    end
```

Comment:
so if we wanted to, we could be passing in the other command line args (like the GCS IP/port)

Reply:
Definitely. I want to tackle that today as a follow up PR